src/constrained_variables.jl

```diff
@@ -6,21 +6,21 @@ function add_constrained_variables(
     single_or_vector_variables_types =
         MOIU.sorted_variable_sets_by_cost(dual_problem.dual_model, primal_model)
     params = Set(variable_parameters)
-    for (F, S) in single_or_vector_variables_types
-        if F === MOI.VectorOfVariables
+    for S in single_or_vector_variables_types
+        if S <: MOI.AbstractVectorSet
```

Comment:
```suggestion
        if F === MOI.VectorOfVariables
```

Reply:
Hi @blegat the problem here is that the function 
```julia
function sorted_variable_sets_by_cost(dest::MOI.ModelLike, src::MOI.ModelLike)
    constraint_types = MOI.get(src, MOI.ListOfConstraintTypesPresent())
    sets = Type[S for (F, S) in constraint_types if _is_variable_function(F)]
    sort!(sets; by = S::Type -> _cost_of_bridging(dest, S))
    return sets
end
```
no longer returns the functions `F`.
I opted to make an if on the sets because they should already be restricted by the function `_is_variable_function(F)`

Reply:
Indeed, you're right, I missed that